msu/hooks/items/item_container.nut

```diff
@@ -72,4 +72,20 @@
 		onNewRound();
 		this.m.ActionCost = ::Const.Tactical.Settings.SwitchItemAPCost;
 	}
+
+	local equip = o.equip;
+	o.equip = function( _item )
+	{
+		local ret = equip(_item);
+		if (ret == true && !::MSU.isNull(this.m.Actor) && this.m.Actor.isAlive()) this.m.Actor.getSkills().onEquip(_item);
+		return ret;
+	}
+
+	local unequip = o.unequip;
+	o.unequip = function( _item )
+	{
+		local ret = unequip(_item);
+		if (ret == true && !::MSU.isNull(this.m.Actor) && this.m.Actor.isAlive()) this.m.Actor.getSkills().onUnequip(_item);
+		return ret;
```

Comment:
onUnequip should be called before the item is unequipped (this is how items handle it). This requires us to redo some checks from item container first, but isn't actually that bad.